### ISSUE TITLE
specify which commands require -n namespace

### ISSUE THREAD

**Issue by grantmiller** [replicatedhq/kots.io#63]
the commands on this page (and their detail pages) should list the ones that require a namespace to run properly https://kots.io/kots-cli/getting-started/

**Comment by austinchambers** [comment]
@grantmiller I made sure to review this when I went through the kots usage. The only command which requires a namespace to run is kots reset-password, and since this is the exception, I made it explicitly required as part of the usage here: https://kots.io/kots-cli/reset-password/

Note how no other command needs the [namespace]

**Comment by austinchambers** [comment]
Verified the CLI documentation also behaves consistently in this way. 

In short: 
* If it's a required argument (like [namespace] for reset-password or [upstream uri] for install, then the command isn't attempted in the CLI and it just displays usage info. 
* I went through all the commands and verified that we get the usage info if and only if one of the usage arguments is missing (not anything in [flags]). 
* This doesn't include situations where, if you have one flag, then you require another. Like some of the special behavior we have for replicated apps where we require -license-file as well.
